README.md

```diff
@@ -81,6 +81,22 @@ To take advantage of Twilio's [Global Infrastructure](https://www.twilio.com/doc
 @client.edge = 'sydney'
 ```
 
+### Enable Debug logging
+
+In order to enable debug logging, pass in a 'logger' instance to the client with the level set to 'DEBUG'
```

Comment:
set to _at least_ 'DEBUG'

Reply:
Debug is the lowest level. So, if they set the log level to anything other than debug or empty, the 'log.debug' statements wouldn't be logged. So, this should be 'set to debug' right?

Reply:
I think Sam's wording is to account for additional levels of logging in future implementations. For example, we may implement INFO level logs at some point.

Reply:
Yes, I was thinking of `TRACE`, but that's not supported by the ruby logger. Either wording is fine with me.